lifecycle/services/export.go

```diff
@@ -0,0 +1,174 @@
+package services
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/httputils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"net/http"
+	"net/url"
+)
+
+const (
+	exportReleaseBundleEndpoint       = "export"
+	exportReleaseBundleStatusEndpoint = "export/status"
+)
+
+type RlExportedStatus string
+
+const (
+	ExportCompleted    RlExportedStatus = "COMPLETED"
+	ExportProcessing   RlExportedStatus = "IN_PROGRESS"
+	ExportFailed       RlExportedStatus = "FAILED"
+	ExportNotTriggered RlExportedStatus = "NOT_TRIGGERED"
+)
+
+type ReleaseBundleExportedStatusResponse struct {
+	Status      RlExportedStatus `json:"status,omitempty"`
+	RelativeUrl string           `json:"relative_download_url,omitempty"`
+	DownloadUrl string           `json:"download_url,omitempty"`
+}
+
+type ReleaseBundleExportParams struct {
+	ReleaseBundleDetails
+	Modifications `json:"modifications"`
+}
+
+type RbExportBody struct {
+	ReleaseBundleExportParams
```

Comment:
If I'm not mistaken, the release bundle details will be sent here although they are not expected to be part of the body.  Better to send only what is expected.